src/Data/String/CodePoints.purs

```diff
@@ -0,0 +1,216 @@
+module Data.String.CodePoints
```

Comment:
These functions allow you to treat PureScript/JavaScript strings as if they were sequences of Unicode code points instead of their true underlying implementation, a sequence of UTF-16 code units.